stdlib/strings/strings.flux

```diff
@@ -1,11 +1,13 @@
 package strings
 
-// The Flux string package provides functions to manipulate UTF-8 encoded strings.
+
+// The Flux strings package provides functions to manipulate UTF-8 encoded strings.
```

Comment:
Can you move this comment to the `package strings` clause? And can you change the first bits to `The strings package ...` from context we know this is a Flux package so we don't need to repeat that for every package description.